spec/features/google_optimize_experiment_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.feature 'Running a GO experiment', js: true do
```

Comment:
I would personally not shorten to GO in the test - I think it adds clarity but open to being persuaded otherwise
```suggestion
RSpec.feature 'Running a Google Optimize experiment', js: true do
```

Reply:
Thanks, Resolved in e5c1f5e